docs/resources/misc/query-vs-reference-formulations.txt

```diff
@@ -0,0 +1,12 @@
+sequenceDiagram
```

Comment:
Is this used somewhere?

Reply:
this is not for me, was already in the resources

Reply:
Hmmmm weird that git then says that this file and the JPEG rendering is added in this PR